utils.py

```diff
@@ -407,3 +407,15 @@ def sortBy(self):
             case "Relevent":
                 sortBy = "sortBy=R"                
         return sortBy
+
+
+def removeSeparators(text: str, separators=['·', '(', '-', '|']) -> str:
+    """Remove text after any of the specified separators and trim whitespace"""
```

Comment:
Instead find a better way of naming the function so it captures the comment content